### ISSUE TITLE
Can't use datepicker in the modal component

### ISSUE THREAD

**Issue by wukaikailive** [vue-bulma/datepicker#47]
I can't use datepicker in the modal component.
Because the modal component will cover the datepicker.

There is my screenshot:
![wpabcszkm 8vaacb0xpn ft](https://cloud.githubusercontent.com/assets/8515824/26402505/45b88dd8-40bd-11e7-8884-9b1d460f831e.png)

Can i get any suggestions?
Thanks.

**Comment by Kuchiriel** [comment]
I have the same issue, I think its is related to flatpickr.

**Comment by luventa** [comment]
related issue. #48 
I will try to find a way to fix this.
Thanks guys!

**Comment by elvisyang2016** [comment]
Any update?

**Comment by Kuchiriel** [comment]
@elvisyang2016 

Try override the css
datepicker {
z-index: 99999;
}

**Comment by havgry** [comment]
I can't replicate the issue locally. The flatpickr z-index is 99999 (999 if `static`) while the Bulma modal is 20. Provide a JSFiddle if you're serious about resolving this.

**Comment by elvisyang2016** [comment]
@Kuchiriel @havgry Thanks. I used to check the issues before I apply it. I do not find this issue too.
But there is another issue blocking me to use it. I wanted add addons to the input dom using slot, but can't get the right style. Finally, I found out this component is based on Bulma 0.3.x, and the addons format already changed in 0.4.x.

**Comment by Kuchiriel** [comment]
@elvisyang2016 Yes, I am using bulma-stylus, which sticks with 0.3.x, It would be interesting to update the components, Bulma 0.4.x looks nice.

**Comment by erythro** [comment]
[Here ](https://github.com/jgthms/bulma/issues/936#issuecomment-517737540) is a css fix for the modal
